### ISSUE TITLE
Different create/load behaviour for netmask

### ISSUE THREAD

**Issue by AlimovSV** [adam12/ecto_network#13]
Any assert in the tests fails when comparing the created entity and the loaded entity
```elixir
iex(5)> ip1 = %IpInfo{} |> IpInfo.changeset(%{ip: {8, 8, 8, 9}}) |> Repo.insert!() |> Map.get(:ip) 
%Postgrex.INET{address: {8, 8, 8, 9}, netmask: 32}
iex(6)> ip2 = IpInfo |> Repo.get_by(%{ip: {8, 8, 8, 9}}) |> Map.get(:ip)
%Postgrex.INET{address: {8, 8, 8, 9}, netmask: nil}
iex(7)> ip1 == ip2
false
```
As you can see the network mask for the newly created IpInfo = 32, but for the loaded is nil (the same for `%{ip: "8.8.8.9"}`)

**Comment by adam12** [comment]
Indeed.

Can you try this branch and see if it fixes your problem? 

https://github.com/adam12/ecto_network/tree/ensure-default-netmask-for-ipv4-inet

**Comment by AlimovSV** [comment]
@adam12 Fantastic! It works for me!

**Comment by AlimovSV** [comment]
@adam12 Thank you very much!

**Comment by adam12** [comment]
Great!

Probably take me a week or two to get out a proper release. You should be OK on this branch for now tho.

Thanks for providing enough information in the initial comment to make debugging easy.

Cheers.

**Comment by adam12** [comment]
Resolved via efcc67aff1d1a9a162186eada4deb740cd2e3719.

**Comment by adam12** [comment]
I just cut [v1.1.0](https://hex.pm/packages/ecto_network/1.1.0) with this fix.

Cheers!
